packages/mold/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/rui314/mold
 TERMUX_PKG_DESCRIPTION="mold: A Modern Linker"
 TERMUX_PKG_LICENSE="AGPL-V3"
```

Comment:
Please [update the license here](https://github.com/rui314/mold/blob/main/LICENSE).